testutil/abci.go

```diff
@@ -0,0 +1,134 @@
+package testutil
+
+import (
+	"time"
+
+	errorsmod "cosmossdk.io/errors"
+	sdkmath "cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	errortypes "github.com/cosmos/cosmos-sdk/types/errors"
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/evmos/evmos/v11/app"
+	"github.com/evmos/evmos/v11/crypto/ethsecp256k1"
+	"github.com/evmos/evmos/v11/encoding"
+	"github.com/evmos/evmos/v11/testutil/tx"
+)
+
+// Commit commits a block at a given time. Reminder: At the end of each
+// Tendermint Consensus round the following methods are run
+//  1. BeginBlock
+//  2. DeliverTx
+//  3. EndBlock
+//  4. Commit
+func Commit(ctx sdk.Context, app *app.Evmos, t time.Duration) sdk.Context {
```

Comment:
I recommend using `TestChain.NextBlock()` from ibc-go as reference since there we also need to add the validator set changes to the new header